diff --git a/hazelbean/utils.py b/hazelbean/utils.py
index 100184b..529ffab 100644
--- a/hazelbean/utils.py
+++ b/hazelbean/utils.py
@@ -1263,9 +1263,9 @@ def df_merge(left_input,
         
         if not supress_warnings:
             if len(comparison_dict['left_only']) > 0:
-                hb.log(f'\nWARNING!\nIn hb.df_merge, left had non-shared values in merge-col. N=: {str(len(comparison_dict['left_only']))}. First few values:' + str(comparison_dict['left_only'])[:100] + '...')
+                hb.log(f"\nWARNING!\nIn hb.df_merge, left had non-shared values in merge-col. N=: {str(len(comparison_dict['left_only']))}. First few values:" + str(comparison_dict['left_only'])[:100] + '...')
             if len(comparison_dict['right_only']) > 0:
-                hb.log(f'\nWARNING!\nIn hb.df_merge, left had non-shared values in merge-col. N=: {str(len(comparison_dict['right_only']))}. First few values:' + str(comparison_dict['right_only'])[:100] + '...')
+                hb.log(f"\nWARNING!\nIn hb.df_merge, left had non-shared values in merge-col. N=: {str(len(comparison_dict['right_only']))}. First few values:" + str(comparison_dict['right_only'])[:100] + '...')
             
     else:
         if left_on == 'index' and right_on == 'index':
